packages/bison.rb

```diff
@@ -33,6 +20,7 @@ def self.install
   end
 
   def self.check
-    system 'make check'
+    puts 'Information: Bison checks take a very long time.'.lightblue
```

Comment:
How about `Notice: ...'.orange`?

Reply:
Isn't orange for warnings? What about green or yellow?

Reply:
This sorta is like a warning.  Yellow would be better than green since that would indicate success.

Reply:
yellow sounds good. Warning indicates something has gone wrong, this is just a notice.

Reply:
4ca78cc